vue3_ts/src/views/veevalidate-yup/CustomComponents.vue

```diff
@@ -0,0 +1,66 @@
+<template>
```

Comment:
A view name has to extend folder name like VeeValidateCustomComponents, to make it clear what file is opened in the IDE. Please check all your views
@AndrewSparrowww please confirm

Reply:
@v-romanyuk It's not always easy to achive as some times your folder name can long like here and having:

VeeValidateCustomComponentsInput - as example.

as for me is too long sometimes.

Ideally yes - we need to extend folder name, but in fact sometimes we can do exception I believe.
Here though, yeap - let's use it as `VeeValidateCustomComponents` - but also think about name, maybe `CustomComponents` is not ideal and doesn't represent what would be there...

cc @m-tarabtsov